### ISSUE TITLE
Please consider removing "Hacker" from branding

### ISSUE THREAD

**Issue by Indigenuity** [philc/vimium#4705]
Vimium is a fantastic plugin for Accessibility purposes, though that matters little in corporate scenarios when every install is audited for security first.  Twice now, I've failed to get auditors to approve the plugin, with the single reason given being "we don't allow hacking tools".

Yes, that's an unreasonable claim to anybody that understands Vimium, but it's perfectly reasonable to someone being introduced to the plugin with the header "The Hacker's Browser"

I see so many places where Vimium is being recommended as the best accessibility tool for folks with poor motor skills, or simply as an ergonomic alternative. It would likely thrive even more if the Hacker branding were toned down.

Thanks for all the wonderful work making this plugin!

**Comment by philg-dev** [comment]
Interesting situation... I guess nobody is ever safe from "corpo BS" anywhere at any time.

In the short term, I can only recommend maybe trying out one of the similar add-ons that are available, like Tridactyl, Vimium-C, etc. If the people making those "security" decisions are dumb enough to turn down something labeled as "hacker's browser", they're probably stubborn and stupid enough to turn down any add-ons like that with no other good reason either, though.

In all fairness though, some features of those add-ons might be considered a "security risk". What comes to mind as an example would be `openCopiedUrlInNewTab` (default `P`) and `openCopiedUrlInCurrentTab` (`default: `p`)... Those are much more likely for novice users to be triggered on accident and there might be sensitive information in the clipboard, which should not be submitted to e.g. the default search engine. I'm not sure if there's any security concerns that are not based on "user error",... But things like that could be fairly reasonable arguments in a corporate setting.

(Those are just my 2 cents, trying to suggest a short-term solution / alternative. I'm just a mere contributor from time to time, so I have no say in the branding-change you've suggested anyway.)
